src/main/java/org/distributed/systems/chord/messaging/FingerTable.java

```diff
@@ -0,0 +1,32 @@
+package org.distributed.systems.chord.messaging;
+
+import org.distributed.systems.chord.model.ChordNode;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class FingerTable {
+
+    public static class Get implements Command, Serializable {
```

Comment:
Kinda weird class name, anyway

Reply:
Inspired by the new behavior API (that we don't use because it's difficult). But this allows you to write FingerTable.Get() what I thought was quite nice looking.